neurons/say/README.md

```diff
@@ -0,0 +1,49 @@
+# shell
+
+## Synopsis
+
+This neuron is the mouth of Kalliope and uses the [TTS](../../Docs/tts.md) to say the given message.
```

Comment:
here I think the link will not works, It should be just "tts.md" as it is placed in the same folder.